course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -419,16 +466,39 @@
        "text": [
-        "<matplotlib.figure.Figure at 0x4705090>"
+        "<matplotlib.figure.Figure at 0x3b75750>"
+       ]
+      }
+     ],
+     "prompt_number": 36
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "# Imshow is similar to a pcolor, but its vertical orientation is reversed (i.e. it would appear upside down compared with a pcolor).\n",
```

Comment:
Again.

This statement is technically true, but misses the detail of why it's happening - that is, imshow takes the input as if it's an image. By default the origin of an image is top left, but the origin of a data plot is bottom left. It's this fact that causes the image to appear upside-down by default, although this can be set with a keyword arg.